tests/test_formats.py

```diff
@@ -744,7 +746,28 @@ def test_copy_update_inference_sites(self):
                 self.assertEqual(list(copy.sites_inference), inference)
                 self.assertEqual(list(data.sites_inference), [True, True, True, True])
 
-    def test_update_inference_sites_bad_data(self):
+    @unittest.skipIf(sys.platform == "win32",
```

Comment:
Now you're using a tempdir the windows problem shouldn't be an issue any more.

Reply:
I'm not sure tempdir solves the problem. I've just tried uncommenting (and changing the file names / dirs to be unique) and it still fails with

```
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process:
```

See https://ci.appveyor.com/project/tskit-dev/tsinfer/builds/28377125

There are a load of other places that use tempdir but are also commented out in the same file, presumably for the same reason.

Reply:
OK, let's not worry about it then. Thanks for checking.

Reply:
Right, dropped the win testing commit I used. This should be ready to merge.